### ISSUE TITLE
Abs for PyTorch via ONNX not supported

### ISSUE THREAD

**Issue by kunaltyagi** [openvinotoolkit/openvino#6002]
##### System information (version)
- OpenVINO => 2020.4
- Operating System / Platform => Ubuntu 20.04
- Problem classification: Model Conversion
- Framework: PyTorch
- Model name: custom

##### Detailed description
Using `torch.abs` results in `compile_tool` unable to compile the ONNX model.

##### Steps to reproduce
https://github.com/kunaltyagi/pytorch_harris/tree/44fc78ad48824e0dd97bba8ee8c6986b11d03dad

##### Issue submission checklist

 - [x] I report the issue, it's not a question
 - [x] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
 - [x] There is reproducer code and related data files: images, videos, models, etc.

**Comment by zulkifli-halim** [comment]
Hello @kunaltyagi can you share with us your model?

**Comment by kunaltyagi** [comment]
```py
class AbsTorch(nn.Module):
	def __init__(self, dtype: torch.dtype = torch.float32):
		self.dtype = dtype

	def forward(self, x):
		return x.abs()

def export_onnx(output_file):
    """
    Exports the model to an ONNX file.
    """
    # Define the expected input shape (dummy input)
    shape = (1, 3, 300, 300)
    # Create the Model
    model = AbsTorch(dtype=torch.float32)
    X = torch.ones(shape, dtype=dtype)
    print(f"Writing to {output_file}")
    torch.onnx.export(
        model,
        X,
        output_file,
        opset_version=12,
        do_constant_folding=True,
    )
```

Do you need a blob, or just a code snippet is ok?

**Comment by zulkifli-halim** [comment]
Yes, send us all the information that can help us to reproduce the issue.

**Comment by kunaltyagi** [comment]
I've attached two files (as txt files, but they are xml and onnx files)

PTAL

[model.xml.txt](https://github.com/openvinotoolkit/openvino/files/6606032/model.xml.txt)
[model.onnx.txt](https://github.com/openvinotoolkit/openvino/files/6606033/model.onnx.txt)

**Comment by zulkifli-halim** [comment]
Hi. I try to generate the .blob file from your model, however ABS does not seem to be supported by MYRIAD/blob.

**Comment by kunaltyagi** [comment]
https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_supported_plugins_Supported_Devices.html#supported_layers

It says Abs is supported for VPU. For a device which has pow, I find it hard to believe there's no hardware support for Abs operation

**Comment by jgespino** [comment]
Hi @kunaltyagi 

Both the compile_tool and benchmark_app show Abs model is not supported by the MYRIAD plugin. Let me verify with the development team if this is the case. 

Regards,
Jesus

Ref. 58801

**Comment by jgespino** [comment]
Hi @kunaltyagi 

I have confirmed with the development team that Abs layer is not supported by the MYRIAD Plugin. I will work with the team to update the documentation. Apologies for the inconvenience.

Regards,
Jesus

**Comment by kunaltyagi** [comment]
Could you relay my question to the development team? Why abs is not supported? Couldn't it be implemented as `pow(pow(x, 2), 0.5)` or a more efficient hardware instruction?

**Comment by jgespino** [comment]
Hi @kunaltyagi 

Here is the response from the development team:
pow(pow(x, 2), 0.5) should work but should not show as good performance as optimized vectorized pre-built instruction on the device side. 

Additionally, a pull request has been started for Abs layer support. You can track the progress  on https://github.com/openvinotoolkit/openvino/pull/6475

Regards,
Jesus

**Comment by kunaltyagi** [comment]
Thanks. A pre built vectorized instruction is exactly what I was looking for

**Comment by zulkifli-halim** [comment]
I'm closing this case. Please feel free to re-open if additional assistance is needed.

**Comment by kunaltyagi** [comment]
I'd have preferred to get this closed by #6475
